doc/controller.md

```diff
@@ -0,0 +1,11 @@
+# PS4 controller documentation
+
+
+
+| Shortcuts    | Descriptions |
+|:-------------|:----------------:|
+| PS+Triangle+L2+R2 | Takes a picture |
```

Comment:
Add this command, to retrieve the picture then use : `scp pi@monkecar.local:/tmp/testcam.jpg /tmp/testcam.jpg && eog /tmp/testcam.jpg`